chatcommands.go

```diff
@@ -269,24 +269,26 @@ var commands = &CommandControl{
 			},
 		},
 
-		common.CNStats.String(): Command{
+		common.CNStats.String(): {
 			HelpText: "Show some stats for stream.",
 			Function: func(cl *Client, args []string) (string, error) {
 				cl.belongsTo.clientsMtx.Lock()
 				users := len(cl.belongsTo.clients)
 				cl.belongsTo.clientsMtx.Unlock()
 
 				// Just print max users and time alive here
-				return fmt.Sprintf("Current users in chat: <b>%d</b><br />Max users in chat: <b>%d</b><br />Server uptime: <b>%s</b><br />Stream uptime: <b>%s</b>",
+				return fmt.Sprintf("Current users in chat: <b>%d</b><br />Max users in chat: <b>%d</b><br />Server uptime: <b>%s</b><br />Stream uptime: <b>%s</b><br />Viewer: <b>%d</b><br />Max Viewer: <b>%d</b>",
```

Comment:
Should probably make "Viewer" and "Max Viewer" plural in the printed text.

Reply:
Oh... Hawk eyeS ^_^